services/services.go

```diff
@@ -88,12 +89,14 @@ func GetProcessConfig(process v3.Process) (*container.Config, *container.HostCon
 	}
 	// var pidMode container.PidMode
 	// pidMode = process.PidMode
+	_, portBindings, _ := nat.ParsePortSpecs(process.Publish)
```

Comment:
Does this method return an error? If so, do not ignore it. You need to return and handle the error if possible.

Reply:
thanks, make senses.